README.md

```diff
@@ -124,7 +124,7 @@ You can configure this storage adapter either through command line options or en
 | Command Line Flag | Environment Variable | Required | Default | Description |
 | --- | --- | --- | --- | --- |
 | `--googleAPIdatasetID` | `PROMBQ_DATASET` | Yes | | Dataset name as shown in GCP |
-| `--googleAPItableID` | `PROMBQ_TABLE` | Yes | | Table name as showon in GCP |
+| `--googleAPItableID` | `PROMBQ_TABLE` | Yes | | Table name as shown in GCP |
 | `--googleAPIjsonkeypath` | `PROMBQ_GCP_JSON` | No | | Path to json keyfile for GCP service account. JSON keyfile also contains project_id. |
 | `--googleProjectID` | `PROMBQ_GCP_PROJECT_ID` | Yes (when -googleAPIjsonkeypath is missing)| | The GCP project_id |
```

Comment:
I think with this change, this is no longer a valid description. Maybe something like this:

```suggestion
| `--googleAPIjsonkeypath` | `PROMBQ_GCP_JSON` | No | | Path to json keyfile for GCP service account. |
| `--googleProjectID` | `PROMBQ_GCP_PROJECT_ID` | No | | The GCP project ID the BigQuery instance is in. If this is not set and `--googleAPIjsonkeypath` is, the project ID in the keyfile will be used |
```

Reply:
I made a change, what do you think?